src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,9 +1,13 @@
+*REMOVED*Microsoft.CodeAnalysis.Compilation.Emit(System.IO.Stream peStream, System.IO.Stream pdbStream = null, System.IO.Stream xmlDocumentationStream = null, System.IO.Stream win32Resources = null, System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.ResourceDescription> manifestResources = null, Microsoft.CodeAnalysis.Emit.EmitOptions options = null, Microsoft.CodeAnalysis.IMethodSymbol debugEntryPoint = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> Microsoft.CodeAnalysis.Emit.EmitResult
```

Comment:
Not unless I've made a mistake. The intent is to follow [Adding Optional Parameters in Public API](https://github.com/dotnet/roslyn/blob/master/docs/Adding%20Optional%20Parameters%20in%20Public%20API.md) by making the existing overload have no optional parameters and creating a new overload with old optional parameters + new optional parameter. It is surface in the txt file as a "removal" of the original overload + the additional of two new ones.